R/readKane.R

```diff
@@ -0,0 +1,63 @@
+#' Raw Read for data corresponding to the Kane dataset
+#'
+#' Download and read in the raw data tables from Kane, E., Ping, C., Chapin, S. and 
+#' Valentine, D. with Bonanza LTER. doi:10.6073/pasta/d09433eee2cb6587eca672864cf7e90f
+#' 
+#' This is not post-processed or QA/QCed by this package.
+#'
+#' @param dataDir filename for download directory
+#' @param download boolean that will download the files from repository 
+#'
+#' @return a list with meta data and raw table reads. This will be 49 KB in size.
+#'
+#' @export
+
+dataDir <- '~/Desktop/Research/Kane' #edit to reflect personal directory
```

Comment:
This needs to be removed. The code in a function file should only contain that function.